### ISSUE TITLE
Is qtranslate-XT still mantained/active? 

### ISSUE THREAD

**Issue by asbestomolesto** [qtranslate/qtranslate-xt#1414]
**Describe the bug**
No updates since months ago

**To Reproduce**
1. try to search for an update
2. find nothing
3. Ask about the plugin state
4. Get zero response from owner

**Expected behavior**
Plugin need to be mantained at least for security reasons

**Additional context**
Is this another case of a translation plugin being abandoned? :(

**Comment by bagaweb** [comment]
I really hope we'll get some news from @herrvigg 

There is also a discussion here: https://github.com/qtranslate/qtranslate-xt/discussions/1410

**Comment by asbestomolesto** [comment]
> I really hope we'll get some news from @herrvigg
> 
> There is also a discussion here: #1410

Yep. I posted here as a bug so MAYBE we can get at least an answer :(

**Comment by rafaelurben** [comment]
It's starting to look like qTranslate-XT will end up like [qTranslate](https://qian.qin.berlin/qtranslate/) and [qTranslate-X](https://github.com/qTranslate-Team/qtranslate-x/issues/579)... Abandoned. 😢 

Many things are already broken, especially due to the lack of full support for the Gutenberg editor.

The new WooCommerce product editor does not work at all with this plugin anymore (#1407), which is a deal breaker for me... I am currently thinking about switching to an alternative translation plugin.

**Comment by bagaweb** [comment]
> Yep. I posted here as a bug so MAYBE we can get at least an answer :(

Yes let's hope to get an answer as soon as possible.

**Comment by bagaweb** [comment]
> I am currently thinking about switching to an alternative translation plugin.

Do you know any similar plugins?

**Comment by asbestomolesto** [comment]
I only know Polylang and we're evaluating it...

**Comment by rafaelurben** [comment]
> Do you know any similar plugins?

I know that WPML and Polylang are popular alternatives, but I haven't analyzed them yet. I know of no similar plugin that stores translations inline and not in duplicate posts/products.

I have seen that WPML has a [migration tool](https://wpml.org/documentation/related-projects/qtranslate-importer/) to migrate away from qTranslate.

**Comment by bagaweb** [comment]
> I know of no similar plugin that stores translations inline and not in duplicate posts/products.

Yes that's my biggest concern, finding a plugin similar to QTX. Personally I don't like the duplicate posts method.

**Comment by rafaelurben** [comment]
> Yes that's my biggest concern, finding a plugin similar to QTX. Personally I don't like the duplicate posts method.

Absolutely! Especially when working with external applications that use the WordPress or WooCommerce API... When pages/products are duplicated, this might have unforseeable consequences.

I'll have to check how Polylang and WPML integrate with the API.

**Comment by herrvigg** [comment]
hi,
sorry there has not been any update for such a long time but I'm still around. I've been very busy and it's been difficult times, unfortunately I had no time to dedicate to freetime projects during the last months.
I could do some basic updates if merge requests are easy to validate. The main problem is some changes require a lot of time to understand by analyzing and testing to make sure it doesn't create new regressions... This is a complex project that would need major refactoring and automated tests. Maybe the community could help by testing the merge requests to make it easier to update.

**Comment by asbestomolesto** [comment]
Happy  for your answer!!! I understand the difficulties. I really hope someone is going to help you. 
Such a big project is very difficult to mantain being alone. <3

**Comment by pozitivepl** [comment]
Great news... glad the author hasn't abandoned this great language plugin :)

**Comment by justinbb** [comment]
> > I know of no similar plugin that stores translations inline and not in duplicate posts/products.
> 
> Yes that's my biggest concern, finding a plugin similar to QTX. Personally I don't like the duplicate posts method.

The similar plugins (with similar syntax for multiplexing languages in a single post or string) are [WP Multilang](https://wordpress.org/plugins/wp-multilang/) and [WPGlobus](https://wordpress.org/plugins/wpglobus/). Around the time qtranslate-XT started up, I tested WPGlobus and found that it was a possible path forward, but qtranslate-XT was a much easier way to keep my site working. I don't recall trying WP Multilang. The relative advantages and disadvantages of each may well have changed over the past six years. (I do not need and have not checked on WooCommerce support.)

WPML and Polylang are of course an entirely different philosophy and require significant database restructuring compared with qtranslate and successors.

**Comment by bagaweb** [comment]
> The similar plugins (with similar syntax for multiplexing languages in a single post or string) are [WP Multilang](https://wordpress.org/plugins/wp-multilang/) and [WPGlobus](https://wordpress.org/plugins/wpglobus/). Around the time qtranslate-XT started up, I tested WPGlobus and found that it was a possible path forward, but qtranslate-XT was a much easier way to keep my site working. I don't recall trying WP Multilang. The relative advantages and disadvantages of each may well have changed over the past six years. (I do not need and have not checked on WooCommerce support.)

I have also checked these plugins a few weeks ago:
- WPGlobus needs a premium extension if you want different slugs per language (same fro WooCommerce);
- WP Multilang hasn't implemented this feature yet.

At the moment QTX is still the best solutions for my needs and I hope Herr Vigg will soon have some time to update it.

**Comment by herrvigg** [comment]
Sorry again I'm still in a very busy situation giving little time for side projects but just to let you know I'm still around. I'll try to do some minimal updates for "low-hanging fruits".

**Comment by herrvigg** [comment]
I will push a 3.15.3 for maintenance fixes soon.
